setup.py

```diff
@@ -83,7 +84,6 @@
         # add them to `docs/requirements.txt`
         'contextlib2',
         'graphviz>=0.8',
-        'numpy>=1.7',
```

Comment:
We could have removed this if all pytorch builds shipped with numpy. The pytorch pip wheel does not install numpy, but will throw an ImportError without numpy, because there are parts of torch that call numpy. I don't see any harm in keeping this in the requirement since it should anyways be installed with pytorch, and our minimum version requirement is an old one.

Reply:
Got it, thanks @neerajprad !

Reply:
It's probably worth adding a comment explaining why we need the numpy requirement, since a code comment is more discoverable than a github comment :wink: